app/src/main/java/com/kks/nimblesurveyjetpackcompose/viewmodel/login/LoginViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.kks.nimblesurveyjetpackcompose.viewmodel.login
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.kks.nimblesurveyjetpackcompose.di.IoDispatcher
+import com.kks.nimblesurveyjetpackcompose.repo.login.LoginRepo
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import javax.inject.Inject
+
+@HiltViewModel
+class LoginViewModel @Inject constructor(
+    private val loginRepo: LoginRepo,
+    @IoDispatcher private val ioDispatcher: CoroutineDispatcher,
+) : ViewModel() {
+
+    fun login() {
+        viewModelScope.launch(ioDispatcher) {
+            val test = loginRepo.loginWithEmailAndPassword("kaung@nimblehq.co", "12345678")
+            test.collect {
+                Timber.d(it.toString())
+            }
```

Comment:
These should definitely _not_ be here in our codebase, we should write a test class for `LoginRepo` and test there